--- linux-5.15.184.orig/Makefile	2020-08-27 03:31:49.000000000 -0400
+++ linux-5.15.184/Makefile	2020-08-28 19:00:50.980172582 -0400
@@ -1351,7 +1351,7 @@
 hdr-inst := -f $(srctree)/scripts/Makefile.headersinst obj
 
 PHONY += headers
-headers: $(version_h) scripts_unifdef uapi-asm-generic archheaders archscripts
+headers: $(version_h) scripts_unifdef uapi-asm-generic archheaders
 	$(if $(filter um, $(SRCARCH)), $(error Headers not exportable for UML))
 	$(Q)$(MAKE) $(hdr-inst)=include/uapi
 	$(Q)$(MAKE) $(hdr-inst)=arch/$(SRCARCH)/include/uapi
